### DISCUSSION TITLE
关闭v2rayu就没有网

### DISCUSSION THREAD

**Discussion by oddhw** [yanue/V2rayU#1318] [Q&A]
必须开启，全局和pac都可以
关闭就整个电脑都没网

**Comment by kun321** [comment]
同样出现过, 可以试试重新连接WiFi或者网线

**Comment by lc6chang** [comment]
可以看看这个comment是不是类似的问题 https://github.com/yanue/V2rayU/issues/1317#issuecomment-1718832330 @solo1d 我之前也是这个问题，然后切换到 `Manual Mode` -> `Turn v2ray-core Off` -> 切回 `Global Mode` 然后就神奇的好了😂，有点玄学，可以试一下。
